CRM/Core/Payment.php

```diff
@@ -261,7 +261,15 @@ public static function logPaymentNotification($params) {
     }
 
     $log = new CRM_Utils_SystemLogger();
-    $log->alert($message, $_REQUEST);
```

Comment:
@seamuslee001 can we put this separately

Reply:
@eileenmcnaughton yes accidentally committed because I was also testing out a pr of Jon’s

Reply:
Done